### PR TITLE
Add quality assurance handbook.

### PR DESCRIPTION
**Pull request by AlexCatch**
Made Web.md into a folder and split out its contents into appropriate. markdown files, I also converted the web quality assurance handbook to a markdown format.

### CODE REVIEW

File: platforms/web/quality-assurance.md [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+- Ensure appropriate values are cached where appropriate.
+
+### Continuous Integration
+
+- Is used, even if it's just a basic setup
+- If Travis CI is being used, ensure third party API keys are input directly into Travis CI’s website relevant to the project and make sure “Display value in build log” is not checked.
+
+
+
+
```
Comment (dextermb) [inline platforms/web/quality-assurance.md]: There's a lot of empty lines here

Comment (AlexCatch) [reply platforms/web/quality-assurance.md]: This should be fixed in the latest commit 👍

File: platforms/web/quality-assurance.md [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+### Forms
+
+- Invalid inputs correctly trigger validation errors
+- All form inputs have associated labels
+- Appropriate validation for different inputs (phone number, postcode)
+- The forms are not too long & and are grouped logically
+- The user is alerted of the form lifecycle (success, error)
+- When modals are used in the context of updates, ensure inputs are correctly pre populated.
+- Autofill is disabled on inputs where appropriate
+- Inputs have correct type property ( emails have email type, phone numbers have tel type)
```
Comment (dextermb) [inline platforms/web/quality-assurance.md]: There shouldn't be a space after the first bracket, right?

Comment (dextermb) [reply platforms/web/quality-assurance.md]: For line 33

**Comment by AlexCatch** [comment]
@dextermb This was pushed before the major restructure, I'll update it & make another PR.

**Pull request closed without merging**